config/initializers/gds-sso.rb

```diff
@@ -0,0 +1,10 @@
+GDS::SSO.config do |config|
+  config.user_model   = 'User'
+
+  # set up ID and Secret in a way which doesn't require it to be checked in to source control...
+  config.oauth_id     = ENV['OAUTH_ID'] || "abcdefg"
+  config.oauth_secret = ENV['OAUTH_SECRET'] || "secret"
+
+  # optional config for location of signonotron2
+  config.oauth_root_url = "http://localhost:3016"
```

Comment:
We use [plek](https://github.com/alphagov/plek) to construct hostnames for our applications, so this line should be `config.oauth_root_url = Plek.find("signon")`.